doc/user/content/sql/create-source/text-s3.md

```diff
@@ -50,7 +50,17 @@ content:
 
 ### Example TEXT format
 
-First we'll create a source that just ingests all these logs, so that we can execute some
+First we'll create a source that ingests a single file. Because we're only ingesting a
+single file, we only require the `ListObject` [permission](https://materialize.com/docs/sql/create-source/text-s3/#permissions-required):
```

Comment:
This interrupts the flow sort of weirdly, IMO. There's a narrative that starts from the listing of the files in the bucket and flows through the various sources; seems weird to have this bit about IAM permissions in the middle. Can we break out to a separate "single object" example at the end of the doc instead?